src/services/rfqm_service.ts

```diff
@@ -424,6 +421,16 @@ export class RfqmService {
         };
     }
 
+    /**
+     * Process an orderHash as an RfqmJob. Throws an error if job must be retried
+     */
+    // tslint:disable: prefer-function-over-method
+    public async processRfqmJobAsync(orderHash: string): Promise<void> {
+        logger.info({ orderHash }, 'finished processing job');
+        // tslint:disable-next-line: custom-no-magic-numbers
+        return Math.random() > 0.5 ? Promise.reject('failing') : Promise.resolve();
```

Comment:
What does the random logic do?

Reply:
this is temporary logic that Daniel and I have added for testing purposes. Essentially, this returns rejects or resolves this flow with 50/50 probability. It allows us to see both the successful and failure code paths.

Will be replaced with an actual impl in the next diff